Added section on using texture packers (#467)
playcanvas/developer.playcanvas.com: content/en/user-manual/2D/texture-packing.md

---
title: Using Texture Packers
layout: usermanual-page.hbs
position: 3
---

# What is texture packing?

> It's common to find sprites or UI images in packs where they are separate images. Texture packing is combining those separate images into a single [texture atlas][texture-atlas].

Reviewer: ```suggestion
It's common to find sprites or UI images bundled as separate images. Texture packing is combining those separate images into a single [texture atlas][texture-atlas].
```

This has several advantages including:

- Faster loading times as it's a single network request instead of many.
- As it's a single texture, the sprites can be batched into a single draw call.

# Tools

> Here are some tools that can be used for texture packing supports PlayCanvas.

Reviewer: ```suggestion
Here are some texture packing tools that are compatible with PlayCanvas.
```

## TexturePacker Online (free)

([Website][texture-packer-online])

A free browser tool that is able to do the basics of texture packing.

![][texture-packer-online-steps]

1. Clear the texture atlas.
2. Add your sprites/UI images.
3. Data format should be kept as JSON hash.
4. Download .png for the texture atlas.
5. Download .json for the frame data.

## Texture Packer tool

([Website][texture-packer-tool])

> TexturePacker is a paid tool for download that has many more features options to create texture atlases with. Including more control over the layout of the sprites/UI images and being able to set up [9 slicing][9-slicing].

Reviewer: ```suggestion
TexturePacker is a paid tool for download that has many more features and options for creating texture atlases. These include more control over the layout of the sprites/UI images and being able to set up [9 slicing][9-slicing].
```
